src/gateways/internal-api.ts

```diff
@@ -13,13 +14,18 @@ export class InternalApiGateway {
     // this.token = token;
   }
 
-  async getEvidenceRequests(): Promise<EvidenceRequest[]> {
-    await sleep();
+  async getEvidenceRequests(rapid?: boolean): Promise<EvidenceRequest[]> {
+    if (!rapid) await sleep();
```

Comment:
Seeing as these sleeps are very temporary, I don't love changing the function signature of this method. Is this needed?

Reply:
no, but the test will include the sleep time otherwise, hmm? unless that's important to test

Reply:
The test including the sleep time isn't the end of the world, it's 1/10th of a second. But it's a moot point anyway because of https://github.com/LBHackney-IT/document-evidence-store-frontend/pull/5#discussion_r540158205 (I realised why you had put this in after submitting my original review 🙈 )